sentry-pureconfig/src/main/scala/com/avast/sst/sentry/pureconfig/implicits.scala

```diff
@@ -0,0 +1,20 @@
+package com.avast.sst.sentry.pureconfig
+
+import pureconfig.ConfigFieldMapping
+import pureconfig.generic.ProductHint
+
+/** Contains [[pureconfig.ConfigReader]] instances with default "kebab-case" naming convention. */
+object implicits extends ConfigReaders {
+
+  /** Contains [[pureconfig.ConfigReader]] instances with "kebab-case" naming convention.
+    *
+    * This is alias for the default `implicits._` import.
+    */
+  object KebabCase extends ConfigReaders
```

Comment:
I don't see any usage of `KebabCase` - is it ok?

Reply:
Yes, this is for the users if they want to use that naming convention, it's not used in SST.